src/Domain/ContentRenderer/CodeRenderer.php

```diff
@@ -47,4 +47,22 @@ private function getLanguageClasses(): array {
 		return [ 'language-' . htmlspecialchars( $this->language ) ];
 	}
 
+	/**
+	 * @return (string|string[])[]
+	 */
+	private function getWrapperAttributes( string $contentUrl ): array {
+		$attributes = [];
+
+		$attributes['class'] = $this->getWrapperClasses();
+
+		$attributes['data-toolbar-order'] = 'copy-to-clipboard';
+
+		if ( parse_url( $contentUrl, PHP_URL_HOST ) == 'bitbucket' ) {
```

Comment:
This isn't going to work for self-hosted bitbucket. Showing the edit button only when the `#bitbucket` function is used is sufficient.

Reply:
Perhaps we can do it like this:
* Pass into that we are dealing with bitbucket to https://github.com/ProfessionalWiki/ExternalContent/blob/master/src/EntryPoints/BitbucketFunction.php#L31 by adding it to the request model
* Pass it from the request model to the render config https://github.com/ProfessionalWiki/ExternalContent/blob/master/src/UseCases/Embed/EmbedUseCase.php#L75
* Use the new render config in the render factory by passing it to the CodeRenderer.

That isn't elegant but I cannot think of a better approach. @malberts maybe you do?

Reply:
Done through the suggested method. Have used an EntryPoint variable instead of isBitBucket, as we might later also be able to use it for anything we need to do for Embed.